### ISSUE TITLE
Display syllable and links between word

### ISSUE THREAD

**Issue by Kamigaku** [espeak-ng/espeak-ng#1820]
Hi everyone, 

I'm trying to find a tool/library that could help me display as phonetics specific sentences.
I'm trying to display, in french, the current sentences:

"J'aime les animaux" and the word "constitution"

The first sentence is displayed as follow:

`Z'Em le-z2 anim'o`

Accordingly to this: https://en.wikipedia.org/wiki/Help:IPA/French the symbol `‿` is used to describe the bound between two words.
I assume that the `z2` part correspond to this symbol as there is a bound between the word `les` and `animaux`. 
Can you confirm that? If yes, is there a way to see association between symbols from the link and character used in espeak?

Secondly, in the word `constitution` it's displayed as follow:

`kO~stitysj'O~`

When, from a dictionnary, it is displayed as followed: `kɔ̃.sti.ty.sjɔ̃`
Once again, i assume that they are representation for special characters but my point is, how does syllable are displayed? Is there an argument to add in the command line to display them?

Thanks!

**Comment by jaacoppi** [comment]
Espeak-ng uses Kirshenbaum by default. Looks like you want IPA notation.

$ espeak-ng -v fr --ipa "J'aime les animaux. Constitution."
ʒˈɛm le-z animˈo
kɔ̃stitysjˈɔ̃

**Comment by Kamigaku** [comment]
> Espeak-ng uses Kirshenbaum by default. Looks like you want IPA notation. $ espeak-ng -v fr --ipa "J'aime les animaux. Constitution." ʒˈɛm le-z animˈo kɔ̃stitysjˈɔ̃

Thanks for your answer, regardless of this, it still does not display the symbol for links between words `‿` or even the syllable with the symbol `.`
Result for `constitution` should be `kɔ̃.sti.ty.sjɔ̃` not `kɔ̃stitysjˈɔ̃` as you mentionned it.

**Comment by jaacoppi** [comment]
You're right. There's also --sep, but it doesn't solve your problem either:

$ espeak-ng -v fr --ipa  "constitution" -x --sep=.
k.ɔ̃.s.t.i.t.y.s.j.ˈɔ̃


Looks like your use case is not supported at the moment.
